.github/workflows/release_portable_linux_pytorch_wheels.yml

```diff
@@ -86,10 +86,12 @@ jobs:
       fail-fast: false
       matrix:
         python_version: ["3.11", "3.12", "3.13"]
-        pytorch_version: ["release/2.7", "release/2.9", "nightly"]
+        pytorch_version: ["release/2.7", "release/2.8", "release/2.9", "nightly"]
```

Comment:
Please update the docs too: https://github.com/ROCm/TheRock/tree/main/external-builds/pytorch#supported-pytorch-versions